src/utils/hash.ts

```diff
@@ -0,0 +1,20 @@
+// https://stackoverflow.com/a/52171480
+export default function hash(str: string, seed = 0): number {
```

Comment:
I find this a bit exaggerated, but it definitely works and gives an unique hash :) Something far simpler would have been enough in this case, but its ok to use this ofc

Reply:
Copy pasted from liikunta-project :)